### DISCUSSION TITLE
How to set Rootster 604 pipe to extend earlier & ZKIVSE to extend higher

### DISCUSSION THREAD

**Discussion by camcombatman** [Courseplay/Courseplay_FS22#3401] [Q&A]
I am experiencing a bit of an issue with vegetable and beet harvesters at the moment. 

What I would like to know is how does one set the ZKIVSE unloading pipe to by default open as high as it will go? No matter what tractor I use, it hits the pipe and causes unloading issues. 

Similarly, for the Rootster 604, the pipe opens too late and as a tractor is pulling alongside which nearly flips the tractor and trailer. 

Anyone know where or how to change the settings for these vehicles to get unload pipes to behave as needed? 

Thanks.

**Comment by Tensuko** [comment]
No setting that can be changed.
I don't know the model Names, but the self driven Harvester works fine with low trailer and mid tier tractors.
The pulled ones rly need small tractors and I could get it working without problems.
Pipe opens too late I have no Idea I did not have this happen.

The open pipe area btw comes default from Giants, you may ask them why they made it that small and not at an offset :)

**Comment by BigBoyDe** [comment]
I Think from version 7.4.1 ZKIVSE is supported. With not to high Tractor and Trailers.
